mappings/net/minecraft/network/packet/s2c/config/ReadyS2CPacket.mapping

```diff
@@ -1 +1,3 @@
 CLASS net/minecraft/class_8733 net/minecraft/network/packet/s2c/config/ReadyS2CPacket
+	FIELD field_48690 INSTANCE Lnet/minecraft/class_8733;
+	FIELD field_48691 ODEC Lnet/minecraft/class_9139;
```

Comment:
```suggestion
	FIELD field_48691 CODEC Lnet/minecraft/class_9139;
```

Reply:
This one's caused by me using a macro on the packet codecs and it sometimes misfireing